docs/3-deployment/deployment-on-heroku.rst

```diff
@@ -58,6 +66,11 @@ Run these commands to deploy the project to Heroku:
 Notes
 -----
 
+Redis Configuration
++++++++++++++++++++
+
+Heroku Redis now requires TLS connections by default. The script above includes the necessary TLS configuration. These settings are optional and default to False, so they won't affect existing deployments unless explicitly enabled.
```

Comment:
No need for this. While it seems like an important information right now, in month+ time, it will be irrelevant, so I'd rather not have this part in our docs